src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerAgreementOrchestrator.cs

```diff
@@ -87,7 +87,7 @@ protected EmployerAgreementOrchestrator()
         }
 
 
-        public async Task<OrchestratorResponse<EmployerAgreementViewModel>> GetById(
+        public virtual async Task<OrchestratorResponse<EmployerAgreementViewModel>> GetById(
```

Comment:
Bit of a smell around having to make methods virtual just to test them, but I can see why that is and a refactor job to tidy up.